lumigator/python/mzai/backend/backend/tests/conftest.py

```diff
@@ -107,40 +111,79 @@ def db_session(db_engine: Engine):
             session.rollback()
 
 
-@pytest.fixture(scope="session")
-def localstack_container():
-    """Initialize a LocalStack test container."""
-    with LocalStackContainer("localstack/localstack:3.4.0") as localstack:
-        yield localstack
+# @pytest.fixture(scope="session")
+# def localstack_container():
+#     """Initialize a LocalStack test container."""
+#     with LocalStackContainer("localstack/localstack:3.4.0") as localstack:
+#         yield localstack
 
 
-@pytest.fixture(scope="session", autouse=True)
-def setup_aws(localstack_container: LocalStackContainer):
-    """Setup env vars/AWS resources for use with the LocalStack container."""
+@pytest.fixture(scope="function")
+def fake_s3_storage() -> dict:
+    return dict()
```

Comment:
should be just `return dict`, we're not doing a method call here, and if we are not doing anything specific to the dict itself, we can call a generic dict without the function in our tests

Reply:
I'd recommend making this field part of the `fake_s3fs(fake_s3_storage)` class

Reply:
We'd be returning a fresh dict here ({} would be clearer, probably).
In any case, it was meant to provide a common storage for the S3 client and the S3FS. But we can just get `MemoryFileSystem.store` and use it. I'll try that out.